### DISCUSSION TITLE
各位github账号都是干嘛的

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#2645] [讨论]
我账号除了fork了PCL的存储库之外，真就只剩两个自己写的垃圾repo了……
你们的不会也是这样的吧……

**Comment by Pigeon0v0** [comment]
你点开别人的看看不就知道了

**Comment by SuoHouDaiShi** [comment]
寄

**Comment by so1ve** [comment]
写代码
![](https://github.com/Hex-Dragon/PCL2/assets/58381667/567c8415-2a5e-4b61-9092-1e39299299b1)

**Reply by iPodtouch6557** [reply]
~联合国利比里亚特派团到现在也不能启动Minecraft真可惜~

**Comment by kong-que-dnf123** [comment]
~我的账号就是单纯在PCL里划水的（）~

**Reply by HomoobyDX** [reply]
我账号基本用来摸鱼的·乐

**Comment by MoYuan-CN** [comment]
开源 开源 开源 凡是自己做啥都想开源（迫真

**Comment by PlayMcBKuwu** [comment]
~~[很明显，更新龙猫和 PCL2 的血泪史](https://github.com/Hex-Dragon/PCL2/discussions/2132)~~
搞个人主页和很多 Minecraft 项目 ~~（指外挂）~~ 以及 [ChatGPT 网页项目](https://github.com/zhile-io/pandora)的 Forks（

**Comment by wxjzx682** [comment]
之前有意向上传代码，但是仔细想想觉得还是不要再GitHub里拉屎了（雾）

**Comment by E0x72-21** [comment]
~~当写issue仙人~~
